### ISSUE TITLE
Make compilation output more verbose

### ISSUE THREAD

**Issue by alcuadrado** [nomiclabs/buidler#197]
Now it's hard to tell if anything is happening.

**Comment by alcuadrado** [comment]
There are some cases (using node 11), where the output is this one

```
% npx buidler compile --show-stack-traces
Compiling...
(node:90888) V8: /private/tmp/bug-fran/cache/compilers/soljson-v0.4.25+commit.59dbf8f1.js:3 Invalid asm.js: Invalid member of stdlib
```

This looks like a failure, but it's not.

**Comment by alcuadrado** [comment]
Fixed by #261
